.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: go
+
+sudo: required
+
+go:
+  - 1.10.x
+
+env:
+  global:
+    - OPERATOR_NAME=apicurio-operator
+
+stages:
+  - test
+  - compile
+  - name: push
+    if: fork = false
+
+jobs:
+  include:
+    - stage: test
+      script:
+      - cd $HOME/gopath/src/github.com/integr8ly/$OPERATOR_NAME && make travis/setup test/unit
+    - stage: compile
+      script:
+      - mkdir -p build/_output/res
+      - cp -r res build/_output/res
```

Comment:
Whats happening here? Whatever this is should it not be done in the makefile?

Reply:
I tried to do it in the makefile and it wouldn't work, it was pretty confusing

Reply:
@odra Why is this needed? Is this to do with copying the template in the Dockerfile? Did we not change this to use packr instead?

Reply:
I've pushed it with the changes back in the Makefile, to see how it fails again